website/blog/2022-10-18-how-to-design-metrics.md

```diff
@@ -0,0 +1,367 @@
+---
+title: "How to design metrics: deciding what to model in the modeling layer vs the semantic layer"
```

Comment:
Love the way we incorporated the thesis in the title! This is a bit wordy - perhaps something along the lines of "How to design metrics: deciding which logic lives modeling layer vs the semantic layer"